src/main/java/com/parasoft/demoapp/model/global/preferences/GlobalPreferencesEntity.java

```diff
@@ -93,6 +93,14 @@ public class GlobalPreferencesEntity {
     @Column(name = "order_service_reply_to_queue")
     private String orderServiceReplyToQueue;
 
+    @Setter
+    @Column(name = "order_service_request")
+    private String orderServiceRequest;
```

Comment:
Rename it to `orderServiceRequestTopic` and rename `orderServiceResponse` to  `orderServiceResponseTopic` , the  column names as well.